client-v2/src/components/FrontsCAPIInterface/SearchInput.js

```diff
@@ -42,47 +89,140 @@ class FrontsCAPISearchInput extends React.Component<
     });
   };
 
-  handleTagInput = (item: any) => {
+  handleTagSearchInput = (
+    { currentTarget }: SyntheticEvent<HTMLInputElement>,
+    type: string
+  ) => {
+    const termInState = `${type}SearchTerm`;
```

Comment:
Could we change the name spacing in the state to `{ searchTerms: { tags: '', sections: '' } } to make this a bit clearer rather than having to create this string?

Reply:
This is pretty ugly but I've been struggling to find a nice simple solution to this. We are already saving the actual selected tags and sections as `tags` and `sections` in state but we need to also keep track of the search terms we use to search capi for tags. I guess we could save these in a separate object in the state? 

```
{ 
  tags: [], 
  sections: [], 
  searchTerms: {
    tags: '', 
    sections
  }
};

```

Reply:
I think that would make it a touch easier to read for me - it just seems like the more robust way to namespace it?